asyncpg/connection.py

```diff
@@ -1231,6 +1236,47 @@ def _drop_global_statement_cache(self):
         else:
             self._drop_local_statement_cache()
 
+    def drop_cache(self):
```

Comment:
This method should be an `async def` method, because we want it to be future-proof.

Reply:
OK, I understand the requirement, I'll push it with a new name.